base/docs/basedocs.jl

```diff
@@ -2529,9 +2529,14 @@ true
 julia> "Hello!" isa IntOrString
 true
 
-julia> 1.0 isa IntOrString
+julia> 1.0 isa IntOrString # Float is neither Int, nor AbstractString
 false
+
+julia> IntOrString(1) # error, cannot instantiate a Union type
+ERROR: MethodError: no method matching Union{Int64, AbstractString}(::Int64)
 ```
+
+**Note:** The empty union [`Union{}`](@ref) is the bottom type of Julia.
```

Comment:
I suspect no one who is looking up `Union` is going to leave feeling informed that it's "the bottom type of julia", *but* I also don't see any harm in including that info. I would suggest a mention of `Base.Bottom` while we're at it though. Something like
```
`Base.Bottom` is an alias for the empty union `Union{}.`
```
An explanation of what the "bottom type" means is probably warranted if this is going to be mentioned at all:
```
The empty union [`Union{}`](@ref) is the bottom type of Julia, meaning it is defined that `x isa Union{} == false` for any `x`.  ### optional additional note: As a consequence. `Union{} <: T == true` for any type `T`.
```

I think it could be useful to throw in this note from the docs too:
```
Unlike most other parametric types, Unions are covariant in their parameters. I.e. `Union{Real, String}` is a subtype of `Union{Number, AbstractString}`.
```
This is also where I would put
```
Like other abstract types, `Union`s cannot be instantiated. 
```

Reply:
> feeling informed that it's "the bottom type of julia"

Do you know any Type theorist? :stuck_out_tongue_closed_eyes:

Jokes aside, all the feedback is really good, let me update this. :D

Reply:
Moved to the extended help.